e2e/suite.go

```diff
@@ -145,22 +147,23 @@ func Run(t *testing.T) {
 	// RunE2ETests by functionality
 
 	suites := map[string]func(*testing.T){
-		"KEY":        singularitykey.RunE2ETests(testenv),
```

Comment:
I do the same mistake once in a while: forget to remove these that I set for testing ;)